### DISCUSSION TITLE
HTML5 Canvas and native Window Mode?

### DISCUSSION THREAD

**Discussion by jhonmolano09** [neutralinojs/neutralinojs#857] [Q&A]
Hello.

Its neutralinojs Native Window or Chrome Mode compatible with HTML5 Canvas?

I have a react app that renders a HTML5 Canvas. 

When i run the app with   Neutralinojs Browser Mode (so it opens in my browser) , it renders the canvas and it works well.

 But if i switch to the window or chrome mode,  it renders the app, but it doesn't render the html5 canvas, instead the <div> element that contains the canvas appears empty.

I'm using Windows 10, React, Chakra Ui

The image below is the app runing on my browser.
![browserneutralino](https://user-images.githubusercontent.com/85649812/162817496-2713cc78-07a2-49bb-8a2c-e61fd1cafbc2.png)

Here is running with neutralino window mode.
![windowsMode](https://user-images.githubusercontent.com/85649812/162817783-d59de3fe-68e1-4717-be3a-b74989261480.png)

And here with chrome mode:
![chromemode](https://user-images.githubusercontent.com/85649812/162817908-2773f99d-f1f5-4437-9e28-123d775fd524.png)

**Comment by schmeling65** [comment]
Bump.

I am also trying to get a HTML5 canvas through some node modules, but beside everything working without an error, It shows nothing except the black background I've set my body to.
Are there any updates since the issue got opened?
